addons/knobs/src/components/types/Color.js

```diff
@@ -84,7 +84,19 @@ class ColorType extends React.Component {
                 this.popover = e;
               }}
             >
-              <SketchPicker color={knob.value} onChange={color => onChange(color.hex)} />
+              <SketchPicker
+                color={knob.value}
+                onChange={color =>
+                  onChange(`
```

Comment:
This code looks nice, but it generates a huge section of noise in the url:

Reply:
https://deploy-preview-2624--storybooks-official.netlify.com/?knob-Dollars=12.5&knob-Name=Storyteller&knob-Years%20in%20NY=9&knob-background=%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20rgba%28%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%2072%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20133%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20151%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%200.14%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%29%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&knob-Age=70&knob-Items%5B0%5D=Laptop&knob-Items%5B1%5D=Book&knob-Items%5B2%5D=Whiskey&knob-Birthday=1484866800000&knob-Nice=true&knob-Styles=%7B%22border%22%3A%223px%20solid%20%23ff00ff%22%2C%22padding%22%3A%2210px%22%7D&knob-Fruit=apple&selectedKind=Addon%20Knobs.withKnobs&selectedStory=tweaks%20static%20values&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

Reply:
Can you solve this somehow?

Reply:
I think this is a formatting issue, since I used a template string and applied prettier to make sure it doesn't exceed the line length (which added newlines). What would you recommend in order to stay within storybook's style guide?

Reply:
You can ignore the problematic eslint rule for this line.

Reply:
@Primigenus that's weird, prettier shouldn't insert newlines into text, only into interpolations. And you actually don't have to run it by hand, it runs automatically as precommit hook